src/server/game/Movement/MovementGenerators/ChaseMovementGenerator.cpp

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2008-2018 TrinityCore <https://www.trinitycore.org/>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ChaseMovementGenerator.h"
+#include "Creature.h"
+#include "MoveSpline.h"
+#include "MoveSplineInit.h"
+#include "G3DPosition.hpp"
+
+static bool isMutualChase(Unit* owner, Unit* target)
```

Comment:
naming sitll, UpperCamelCase unless u want to change the standard :dancer:

Reply:
not a member function